meseq.py

```diff
@@ -176,8 +178,14 @@ def text(self, x, y, text, flags = ALIGN_CENTER):
         layout.set_alignment(pango.ALIGN_CENTER)
         w, h = layout.get_size()
         w = w / pango.SCALE
-        h = h / pango.SCALE
+        h = h / pango.SCALE + 1
+        if white_bg:
```

Comment:
ok, I saw the example "example_crossing_lifelines.msq", and I get the point.

But there is a problem : the line of the arrow is slightly hidden below the bottom of the white background. To fix that, please modify "def arrow()" so that the text is draw before the arrow.